qiskit/primitives/base_estimator.py

```diff
@@ -100,18 +100,28 @@
         #             <psi1(theta3)|H3|psi1(theta3)> ]
         result5 = e([0, 1, 0], [0, 1, 2], [theta1, theta2, theta3])
         print(result5)
+
+        # It is possible to pass objects.
+        # calculate [ <psi2(theta2)|H2|psi2(theta2)> ]
+        result6 = e([psi2], [H2], [theta2])
+        print(result6)
```

Comment:
Do we need to mention somewhere that using indices have better performance? And do we have a way to benchmark the performance between the two ways? I think it would be good to have an idea. We can add those in a tutorial or how-to guide.

Reply:
Yes. I'll mention the performance.

We can add the performance tests here: https://github.com/Qiskit/qiskit/tree/master/test/benchmarks
Tutorials are in qiskit-ibm-primitives.

Reply:
fc499f0